client-sdk/go/modules/consensusaccounts/types.go

```diff
@@ -199,6 +214,8 @@ type DelegateEvent struct {
 	To     types.Address   `json:"to"`
 	Amount types.BaseUnits `json:"amount"`
 	Error  *ConsensusError `json:"error,omitempty"`
+	// Added in runtime-sdk v0.15.0.
```

Comment:
Will these comments be removed in the future?

Reply:
Not sure. As far as I'm concerned, I can also remove the comment now. But for someone looking at the code without this comment, it won't be obvious why this field is potentially empty.